CHANGELOG.md

```diff
@@ -2,6 +2,78 @@
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [2.2.1](https://github.com/consuldemocracy/consuldemocracy/tree/2.2.1) (2024-09-xx)
+
+[Full Changelog](https://github.com/consuldemocracy/consuldemocracy/compare/2.2.0...2.2.1)
+
+### Added
+
+- **Documentation:** Update user content translations documentation [\#5710](https://github.com/consuldemocracy/consuldemocracy/pull/5710)
+- **Documentation:** Update local census documentation [\#5709](https://github.com/consuldemocracy/consuldemocracy/pull/5709)
+- **Documentation:** Update remote census configuration documentation [\#5708](https://github.com/consuldemocracy/consuldemocracy/pull/5708)
+- **Documentation:** Update OAuth documentation [\#5707](https://github.com/consuldemocracy/consuldemocracy/pull/5707)
+- **Documentation:** Update multitenancy documentation [\#5699](https://github.com/consuldemocracy/consuldemocracy/pull/5699)
+- **Documentation:** Update recommendations documentation [\#5697](https://github.com/consuldemocracy/consuldemocracy/pull/5697)
+- **Documentation:** Update documentation for Production and Staging servers [\#5696](https://github.com/consuldemocracy/consuldemocracy/pull/5696)
+- **Documentation:** Update GraphQL documentation [\#5695](https://github.com/consuldemocracy/consuldemocracy/pull/5695)
+- **Documentation:** Update basic configuration guides [\#5694](https://github.com/consuldemocracy/consuldemocracy/pull/5694)
+- **Documentation:** Update Docker installation documentation [\#5693](https://github.com/consuldemocracy/consuldemocracy/pull/5693)
+- **Documentation:** Update documentation to customize the application [\#5685](https://github.com/consuldemocracy/consuldemocracy/pull/5685)
+- **Documentation:** Update getting started instructions [\#5655](https://github.com/consuldemocracy/consuldemocracy/pull/5655)
+- **Documentation:** Update local installation instructions [\#5652](https://github.com/consuldemocracy/consuldemocracy/pull/5652)
+- **GraphQL:** Add new GraphQL types for budget investments [\#5571](https://github.com/consuldemocracy/consuldemocracy/pull/5571)
+
+### Changed
+
+- **Linters:** Add properties-alphabetical-order Stylelint rule [\#5493](https://github.com/consuldemocracy/consuldemocracy/pull/5493)
+- **Maintenance:** Use dependabot to update github actions dependencies [\#5611](https://github.com/consuldemocracy/consuldemocracy/pull/5611)
+- **Maintenance-CI:** Use a maintained version of PostgreSQL in Docker [\#5691](https://github.com/consuldemocracy/consuldemocracy/pull/5691)
```

Comment:
Since this affects the `docker-compose.yml` file that some people might use in development.

```suggestion
- **Maintenance-Docker:** Use a maintained version of PostgreSQL in Docker [\#5691](https://github.com/consuldemocracy/consuldemocracy/pull/5691)
```